### PR TITLE
89 change article font size

### PR DESCRIPTION
**Pull request by b1s0ndev**
I was wondering what the conversion from px to em was, not too sure where to find out the base em?
thanks

### CODE REVIEW

**Comment by psiemens** [comment]
@sjason19 haha no worries

File: ubyssey/static/src/styles/modules/article/_base.scss [hunk 1]
```diff
@@ -151,8 +151,8 @@ div.article-content {
     font-family: $font-body !important;
     font-size: 17px !important;
```
Comment (psiemens) [inline ubyssey/static/src/styles/modules/article/_base.scss]: We have a SASS helper function that automatically converts `px` to `em`. You can use it like this:

 ```sass
font-size: font-size(17) !important;
```

File: ubyssey/static/src/styles/modules/article/_base.scss [hunk 2]
```diff
@@ -151,8 +151,8 @@ div.article-content {
...
     color: #333333 !important;
-    line-height: 26px !important;
-    margin-top: 20px;
+    line-height: 27px !important;
```
Comment (psiemens) [inline ubyssey/static/src/styles/modules/article/_base.scss]: Change this to use `em` like this:

```sass
line-height: 1.55em !important;
```

File: ubyssey/static/src/styles/modules/article/_base.scss [hunk 3]
```diff
@@ -151,8 +151,8 @@ div.article-content {
...
+    margin-top: 27px;
```
Comment (psiemens) [inline ubyssey/static/src/styles/modules/article/_base.scss]: Let's use `1.4em` here instead of pixels:

```sass
margin-top: 1.4em;
```

**Review by psiemens — CHANGES_REQUESTED** [review]

**Review by b1s0ndev — COMMENTED** [review]
didnt mean to do this sorry

**Review by b1s0ndev — COMMENTED** [review]
let me know if this is all that had to be changed :)

**Comment by psiemens** [comment]
Perfect, thanks @sjason19!

**Review by psiemens — APPROVED** [review]

**Pull request merged**